examples/SDL2/main.cpp

```diff
@@ -0,0 +1,15 @@
+#include<iostream>
+#include<SDL/SDL.h>
```

Comment:
If it helps I checked what its trying to download (https://raw.githubusercontent.com/ingenue/hunter-cache/master/e77cb10/SDL2/2.0.4-h1/632a400/da39a3e/a49b0e5/da39a3e/basic-deps.DONE) and its just says "Not Found"